src/electron/start.action.mjs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from 'url';
+import electron from 'electron';
+
+import {runAction} from '../build/run_action.mjs';
+import {getElectronBuildParameters} from './get_electron_build_parameters.mjs';
+import {getRootDir} from '../build/get_root_dir.mjs';
+import {spawnStream} from '../build/spawn_stream.mjs';
+
+/**
+ * @description Builds and starts the electron application.
+ *
+ * @param {string[]} parameters
+ */
+export async function main(...parameters) {
+  const {platform, buildMode} = getElectronBuildParameters(parameters);
+
+  await runAction('www/build', platform, `--buildMode=${buildMode}`);
+  await runAction('electron/build_main', ...parameters);
+  await runAction('electron/build', platform, `--buildMode=${buildMode}`);
```

Comment:
What about the preload script?

Reply:
@jyyi1 how is that built, with the new change?